README.md

```diff
@@ -5,16 +5,19 @@ We are a [Rails company](https://rubyonrails.org/foundation), so the most value
 will be found in the parts concerning Rails. But anyways you'll also find a lot about the
 inner workings of Renuo.
 
-* [Ruby on Rails – App Setup Guide](./ruby_on_rails/README.md)
+* [Ruby on Rails – Application Setup Guide](./ruby_on_rails/README.md)
 
-## Some Notes on the Side
+## Some Notes on the side
```

Comment:
We seem to use [Title Case](https://en.wikipedia.org/wiki/Title_case) consistently so I'd keep the noun capitalized.
```suggestion
## Some Notes on the Side
```